app/containers/Questions/Content/Body/IPFSInformation.js

```diff
@@ -73,13 +73,19 @@ const IPFSInformation = ({ locale, ipfsHash, histories }) => {
       {formattedData?.length > 0 && (
         <table className="table mt-1 mb-0">
           <thead>
-            {Object.values(columns).map(column => <th>{column}</th>)}
+            <tr>
+              {Object.values(columns).map(column => (
+                <th key={column}>{column}</th>
+              ))}
+            </tr>
           </thead>
 
           <tbody>
             {formattedData.map(item => (
-              <tr>
-                {Object.keys(columns).map(column => <td>{item[column]}</td>)}
+              <tr key={item.transactionHash.props.children}>
+                {Object.keys(columns).map(column => (
+                  <td key={column}>{item[column]}</td>
+                ))}
```

Comment:
I'm not sure if these are ideal values ​​for keys, but in my opinion this is the only correct solution

Reply:
column unique value for each key?

Reply:
done